kaldb/src/main/proto/kaldb_search.proto

```diff
@@ -35,6 +38,17 @@ message HistogramBucket {
   double count = 3;
 }
 
+message Aggregations {
+  repeated Aggregation agg = 1;
+}
+
+message Aggregation {
+  string agg_name = 1;
+  string agg_type = 2;
+  string agg_field = 3;
+  repeated double values = 4;
```

Comment:
Used only for `agg_type = 'percentile_ranks'`

Reply:
What if we use a generic dictionary in request and response instead do the agg_type can customize the values to be sent and received.